lua/nvim-tree/api.lua

```diff
@@ -265,8 +264,18 @@ Api.git.reload = wrap(actions.reloaders.reload_git)
 Api.events.subscribe = events.subscribe
 Api.events.Event = events.Event
 
-Api.live_filter.start = wrap(live_filter.start_filtering)
-Api.live_filter.clear = wrap(live_filter.clear_filter)
+Api.live_filter.start = wrap(function(...)
+  local explorer = core.get_explorer()
+  if explorer then
+    return explorer.live_filter:start_filtering(...)
+  end
+end)
+Api.live_filter.clear = wrap(function(...)
+  local explorer = core.get_explorer()
+  if explorer then
+    return explorer.live_filter:clear_filter(...)
+  end
+end)
```

Comment:
Sorry, I didn't communicate this at all...

There's a somewhat hacky function to do this using member and method names:

```suggestion
Api.live_filter.start = wrap_explorer_member("live_filter", "start_filtering")
Api.live_filter.clear = wrap_explorer_member("live_filter", "clear_filter")
```